docs/overview/deep_dive/ivy_tests.rst

```diff
@@ -249,6 +249,33 @@ One of the few differences between the 4 decorators that they generate different
 
 Our test decorators actually transforms to :code:`@given` decorators at PyTets collecting time, therefore this allows us to use other **Hypothesis** decorators like, :code:`@reproduce_failure`, :code:`@settings`, :code:`@seed`.
 
+- Decorator order
```

Comment:
I think this should be placed on the Function Wrappers page of the deep dive instead (as everyone's already mentioned), as that page also has descriptions of what each decorator does